policies/glossary.md

```diff
@@ -137,6 +156,12 @@ A [patch release] is one where API and ABI breaking changes are **not** permitte
 and no additional functionality can be added.
 Refer to the [versioning policy] for specific details.
 
+## perlasm
+
+Assembly code wrapper written in Perl.  This is done to ease compatibility
+problems across assembler, platforms and processor revisions.
```

Comment:
Nit: Should this be "assemblers" (plural)?

Reply:
Either "assemblers" or "platform", depending whether "revisions" applies only to "processor" or to all three